### ISSUE TITLE
a11y Bug: Cookie dialog missing 'name'  

### ISSUE THREAD

**Issue by a-kosanovic** [ALASdoo/alasdoo_com_v2#203]
- Solution: add `aria-label` or `aria-labeledby` to the `<dialog ` element 
- https://dequeuniversity.com/rules/axe/4.9/aria-dialog-name
> Aria dialog elements must have discernible text that clearly describes the destination, purpose, function, or action for screen reader users.

![image](https://github.com/user-attachments/assets/8f5f7ebe-dca3-4b9d-a932-c07587507199)

- [ ] Layout doesn't match the design
